website/docs/installation/automated-install.md

```diff
@@ -23,3 +23,20 @@ Requires authentik 2023.3
 :::
 
 Set the email address for the default `akadmin` user.
+
+## Kubernetes
+
+Set the `akadmin`user password and token:
+```
+authentik:
+  bootstrap_token: test
+  bootstrap_password: test
+```
+
+To store it in a secret, use:
+```
+envFrom
+ - secretRef:
+     name: some-secret
+```
+where some-secret would contain the environment variables as in the documentation.
```

Comment:
```suggestion
where _some-secret_ contains the environment variables as in the documentation above.
```

Reply:
Hi @Zapfmeister first off all, thank you very much for adding this to our docs; we definitely need more How To/procedural docs. This is a great addition. 

For the last sentence (line 42), can you explain a bit more what that means? Maybe give an example with the exact syntax? Which one of the three variables listed above in the docs would one use for the `some-secret`? Any of them? 

Thanks, I am still learning authentik myself, and I want to make the docs easy for newbies, too.